src/classes/HiFiMixerSession.ts

```diff
@@ -485,9 +485,9 @@ export class HiFiMixerSession {
                 if (instructionName === "mute") {
                     let shouldBeMuted: boolean;
                     if (instructionArguments.length >= 1) {
-                        if (instructionArguments[0] === 1) {
+                        if (instructionArguments[0] === true) {
                             shouldBeMuted = true;
-                        } else if (instructionArguments[0] === 0) {
+                        } else if (instructionArguments[0] === false) {
```

Comment:
Could this perhaps instead be
`shouldBeMuted = instructionArguments[0]`?

Reply:
(Or do we not trust the incoming value of instructionArguments[0]?)

Reply:
We probably do trust the value of `instructionArguments[0]`, but for completeness (and Typescript type correctness), I've simplified the code but still check if it's a boolean.